### DISCUSSION TITLE
Problem With Projectiles

### DISCUSSION THREAD

**Discussion by Parkopalis** [Pryaxis/TShock#2192] [Q&A] [answered]
Console Is Saying "debug : /OnNewProjectile please report to tshock about this! normally this is a reject from (Player) 6
I do not know what caused this but it said to report to tshock so here I am. :D Anyone have a fix?

**Comment by hakusaro** [comment, marked as answer]
Thanks for reporting in! You don't have to do anything, but you can turn

debug logs off in the config (set the debug log option to false).

On December 27, 2020, GitHub <notifications@github.com> wrote:
> Console Is Saying "debug : /OnNewProjectile please report to tshock
> about this! normally this is a reject from (Player) 6
>  I do not know what caused this but it said to report to tshock so
> here I am. :D Anyone have a fix?
>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Pryaxis/TShock/discussions/2192>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAECBJYS43ROOOEVZBDRWXLSW7SMJANCNFSM4VLNFJ3Q>.

**Reply by Parkopalis** [reply]
Thanks! Helps a bunch. Makes me wonder why it says to contact tshock.

**Comment by hakusaro** [comment]
Thanks for your valuable thought. I have cataloged it for future
purposes.

On December 29, 2020, GitHub <notifications@github.com> wrote:
> Thanks! Helps a bunch. Makes me wonder why it says to contact tshock.

>

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/Pryaxis/TShock/discussions/2192#discussioncomment-

251507>, or unsubscribe <https://github.com/notifications/unsubscribe-

auth/AAECBJ3ADICPXQEERKWLNR3SXJS7RANCNFSM4VLNFJ3Q>.
